### DISCUSSION TITLE
Repeat crosshair for a given period

### DISCUSSION THREAD

**Discussion by levdimov** [grafana/grafana#37863] [Dashboards, Panels & Visualizations]
Hello!

I thought that it would be cool and quite convinient to have an opportunity to visually (and more precisely) compare two (several?) points in time on a graph, like this:

![repeated-crosshair](https://user-images.githubusercontent.com/2759370/129310993-04a8759e-5638-4cae-8271-5d0a4000b20b.jpg)

You could specify a period (within a dashboard time range) which the crosshair will be repeated with (1h, 12h, 1d, etc.)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
